### ISSUE TITLE
1.4.2 Normative text is missing a comma, which could potentially cause misinterpretations of the meaning

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3612]
[1.4.2 Audio control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html) reads as follows:
>If any audio on a Web page plays automatically for more than 3 seconds, either a [mechanism](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html#dfn-mechanism) is available to pause or stop the audio, or a mechanism is available to control audio volume independently from the overall system volume level.

Both the ability to pause/stop the audio, and the ability to control the volume are intended to be independent of system volume. This is explicitly stated in the Intent, where user who have screen readers read text to them "can find it hard to hear the speech output if there is other audio playing at the same time". So "it is important that the user be able to turn off the background sound" without turning off the audio output from the screen reader software.

However, because there is no comma between "volume" and "independently", it is possible to misread that the ability to stop/pause is NOT independent of the overall system volume level.

This is easily remedied by the addition of a comma to make the associations correct and clear:
> [1.4.2 Audio control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html) reads as follows:
>If any audio on a Web page plays automatically for more than 3 seconds, either a [mechanism](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html#dfn-mechanism) is available to pause or stop the audio, or a mechanism is available to control audio volume, independently from the overall system volume level.

**Comment by awkawk** [comment]
@mbgower I don't think that the concept of pausing or stopping the audio is related to "overall system volume level". How would "stopping the audio" and "overall system volume level" be connected in a way that causes issues interpretation?

**Comment by patrickhlauke** [comment]
maybe not overall system volume level, but more generall overall system audio output. i.e. you can't say "sure, there's a mechanism to pause/stop the audio ... just turn off your speakers!"

**Comment by GreggVan** [comment]
Right - that would turn off the screen reader   and  all aids for cognitive, language, and learning disabilities that involve speech.

**Comment by cstrobbe** [comment]
I have always read it as follows:

> If any audio on a Web page plays automatically for more than 3 seconds, 
> **(a)** either a [mechanism](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html#dfn-mechanism) is available to pause or stop the audio, 
> **(b)** or a mechanism is available to control audio volume independently from the overall system volume level.

In other words, "independently from the overall system volume level" refers only to the second part. "Pause" or "stop" in the first part refers to audio that is playing in the web content, and pausing and stopping that audio using controls in a player or elsewhere on the page would have no impact on the operating systems audio volume level (and thus no impact on the screen reader's volume level). 

(a) "Pause" assumes that the audio could resume at the same point later and "stop" assumes that the audio might be restarted later, 
(b) whereas "control the audio volume" assumes that the audio continues playing but without interfering with the audio output from a screen reader or from text-to-speech software used by people with specific learning disabilities.

**Comment by patrickhlauke** [comment]
> "Pause" or "stop" in the first part refers to audio that is playing in the web content

well that's the part that perhaps needs clarifying, because it's not explicit that this means "and just in the web content". admittedly a too literal interpretation, but turning off the entire system's audio would technically also be a "mechanism" to "pause or stop the audio" of the content.

**Comment by awkawk** [comment]
> you can't say "sure, there's a mechanism to pause/stop the audio ... just turn off your speakers!"

I don't think that anyone believes (or should believe) that turning off speakers pauses or stops an audio file. 

This seems like it should be clarified in the Understanding document, and it is: "Having control of the volume includes being able to reduce its volume to zero. Muting the system volume is not "pausing or stopping" the autoplay audio. Both the "pause or stop" and control of audio volume need to be independent of the overall system volume."

Where is this causing confusion?
